<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FilterPanelWidget</class>
 <widget class="QFrame" name="FilterPanelWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>42</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Frame</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <item>
    <widget class="QLineEdit" name="filterTracks">
     <property name="accessibleName">
      <string>FilterTracks</string>
     </property>
     <property name="styleSheet">
      <string notr="true">QLineEdit {
	background-image: url(:/actions/filter_list_small_offset);
}</string>
     </property>
     <property name="placeholderText">
      <string>Filter Tracks</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLineEdit" name="filterFunctions">
     <property name="accessibleName">
      <string>FilterFunctions</string>
     </property>
     <property name="placeholderText">
      <string>Filter Names</string>
     </property>
     <property name="clearButtonEnabled">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="label">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="pixmap">
      <pixmap resource="../../icons/orbiticons.qrc">:/actions/access_time</pixmap>
     </property>
     <property name="scaledContents">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="timerLabel">
     <property name="text">
      <string>0s</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../../icons/orbiticons.qrc"/>
 </resources>
 <connections/>
</ui>
